src/psyclone/generator.py

```diff
@@ -21,16 +21,84 @@
 import sys
```

Comment:
Please add your name and copyright information to the top of the file. Note, the license information is actually incorrect but we can change that separately.